doc/sphinx/practical-tools/utilities.rst

```diff
@@ -190,6 +190,85 @@ since it results in a “packed” plugin: All auxiliary modules (as
 (``Qux_plugin``). This reduces the chances of begin unable to load two
 distinct plugins because of a clash in their auxiliary module names.
 
+.. _arbitrary_arguments:
+
+Passing arbitrary arguments to coqc
++++++++++++++++++++++++++++++++++++
+Using either ``-arg "-w all"`` or ``-arg -w -arg all`` its possible to pass the
+option ``-w all`` to ``coqc``. For options like ``-set 'Default Goal
+Selector=!'`` quotes are necessary: ``-arg "-set 'Default Goal Selector=!'"``.
+It is impossible to pass a double quote using ``-arg``, only single quotes work
+in this situation.
+
+.. _comments:
+
+Comments
+++++++++
+The syntax for comments in a ``_CoqProject`` file is a bit unusual. The
+character ``#`` starts a comment until the end of the line only if it is
+outside a quoted string, and preceded by whitespace or a closing double quote.
+This means ``Filename#.v`` and ``"Filename With # Spaces.v"`` won’t cause
+comments. The ``#`` is in these cases treated like an ordinary character.
```

Comment:
Since newline characters are whitespace, if `#` appears at the beginning of the line, it technically is "preceded by whitespace", thus starting a comment.
But I worded it a bit differently, so the text doesn’t have to explain this technicality.

Reply:
(I think `#` as the first character of the file also starts a comment)

Reply:
Yes, I wasn’t too clear above.